pytest_tap/plugin.py

```diff
@@ -20,7 +20,7 @@
 
 def pytest_addoption(parser):
     """Include all the command line options."""
-    group = parser.getgroup('terminal reporting', after='general')
+    group = parser.getgroup('terminal reporting', 'reporting', after='general')
```

Comment:
I checked the pytest code and saw that this is some description that is consistently put in the source code. I'm still not sure why it's necessary, but that's ok.